lisa/core/sshConnection.py

```diff
@@ -1,4 +1,9 @@
-import os
+from typing import Optional
+
+import paramiko
+
+from lisa.util.connectionInfo import ConnectionInfo
+from lisa.util.excutableResult import ExecutableResult
 
 
 class SshConnection:
```

Comment:
I'm wondering if either of the popular SSH libraries already provide this kind of connection abstraction, I'd hate to make our own.

Reply:
packages like to provide rich funtionality, it may needs a lightweight wrapper to simplify... Feel free to try new one.